train.py

```diff
@@ -112,16 +118,30 @@ def main(job_config: JobConfig):
     # 3. max_seq_len base on inputs
     model_config.norm_type = job_config.model.norm_type
     model_config.vocab_size = tokenizer.n_words
+    model_config.vocab_size = 50000
```

Comment:
I don't think we should hardcode this, we will be adding tokens to the base Galactica tokenizer anyways.

Reply:
yeah, this is for successful loading of the weights only

Reply:
Ok, once tokenizers are integrated, we should change this. @Knarik1